.NET/CreateAllPackages.cmd

```diff
@@ -30,7 +30,7 @@ popd
 pushd Microsoft.Recognizers.Text.Choice
 call CreatePackage.cmd
 popd
-pushd Microsoft.Recognizers.Text.DataTypes.TimexExpression
```

Comment:
Please don’t revert other people changes. You should rebase/merge your PR on /master